plugin/init.lua

```diff
@@ -14,9 +14,11 @@ local workspace_formatter = function(label)
 	})
 end
 
+---@param extra_args string
```

Comment:
Can you add a questionamrk to this parameter like so: ---@param extra_args**?** string

This will mark it as optional in the type annotation [see here](https://github.com/LuaLS/lua-language-server/wiki/Annotations)

Reply:
Yep. Fixed both in 2f01168.